### DISCUSSION TITLE
v2.5.1

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#43] [Announcements]
## What's Changed
* fix: add missing handling for cloth category for u2net_cloth_seg
* fix: move current_task_id increase to handler again
* refactor: rename checkbox `Enable Mask Upload` to `Enable Advanced Masking Features`

**Full Changelog**: https://github.com/mashb1t/Fooocus/compare/v2.5.0...v2.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.5.1'>v2.5.1</a>.</em>

**Comment by Snuux** [comment]
thank you for your work!)

**Reply by p1agnut** [reply]
agreed
